crates/containerd-shim-wasmtime/src/executors/container.rs

```diff
@@ -0,0 +1,175 @@
+use libcontainer::workload::{Executor, ExecutorError, EMPTY};
```

Comment:
**thought:** I believe this would better suited in `containerd-shim-wasm` since there's nothing `wasmtime` specific here. That way we can reuse it for `wasmedge`.

Reply:
You are 100% right about it. Once #226 is merged, I will move these to `containerd-shim-wasm`, until then, I don't want to introduce the deps on `libcontainer` yet.

Reply:
Done.